packages/ra-core/src/auth/usePermissions.spec.tsx

```diff
@@ -53,14 +53,10 @@ describe('usePermissions', () => {
             getPermissions: () => Promise.resolve('admin'),
         };
         render(
-            <CoreAdminContext authProvider={authProvider}>
-                <UsePermissions>{stateInpector}</UsePermissions>
-            </CoreAdminContext>
+            <WithAuthProviderAndGetPermissions authProvider={authProvider} />
```

Comment:
this story has a 300ms delay. Make it configurable so the the test is faster than that.

Reply:
That's why I pass an authProvider ;)